### DISCUSSION TITLE
No EXE?

### DISCUSSION THREAD

**Discussion by diskbeta** [FunkinCrew/Funkin#2185] [Compiling help]
after I compiled the source code, I don't get my exe file. also I checked the export/release/windows/bin

**Comment by moxie-coder** [comment]
did you check if there was any errors while compiling?

**Reply by diskbeta** [reply]
yes

**Reply by diskbeta** [reply]
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/Polymod.hx:1005: characters 1-7 : Warning : (WDeprecatedEnumAbstract) `@:enum abstract` is deprecated in favor of `enum abstract`
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/Polymod.hx:1051: characters 1-7 : Warning : (WDeprecatedEnumAbstract) `@:enum abstract` is deprecated in favor of `enum abstract`
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/backends/PolymodAssets.hx:226: characters 1-7 : Warning : (WDeprecatedEnumAbstract) `@:enum abstract` is deprecated in favor of `enum abstract`
C:/HaxeToolkit/haxe/lib/flixel-ui/2,5,0/flixel/addons/ui/FlxUICursor.hx:564: characters 9-11 : Warning : Potential typo detected (expected similar values are flixel.addons.ui.SortMethod.ID)
source/NGio.hx:12: characters 1-68 : Abstract<io.newgrounds.objects.events.Result> has no field GetCurrentVersionResult
source/NGio.hx:13: characters 1-61 : Abstract<io.newgrounds.objects.events.Result> has no field GetVersionResult
source/NGio.hx:47: characters 62-76 : io.newgrounds.Call<io.newgrounds.objects.events.GetCurrentVersionData> has no field addDataHandler (Suggestion: addStatusHandler)
source/NGio.hx:47: characters 104-127 : Type not found : GetCurrentVersionResult
source/TitleState.hx:61: characters 16-28 : Type not found : APIStuff
source/PlayState.hx:55: characters 21-29 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/PlayState.hx:99: characters 17-25 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/PlayState.hx:224: characters 21-29 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/PlayState.hx:988: characters 17-25 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/PlayState.hx:990: characters 17-25 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/PauseSubState.hx:20: characters 17-25 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/PauseSubState.hx:26: characters 20-28 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/ChartingState.hx:81: characters 13-21 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/ChartingState.hx:357: characters 16-24 : Warning : (WDeprecated) flixel.system.FlxSound was moved to flixel.sound.FlxSound
source/NGio.hx:96: characters 25-39 : () -> Void should be Null<io.newgrounds.objects.events.Outcome<io.newgrounds.CallError> -> Void>
source/NGio.hx:96: characters 25-39 : ... For optional function argument 'callback'
source/NGio.hx:99: characters 30-45 : () -> Void should be Null<io.newgrounds.objects.events.Outcome<io.newgrounds.CallError> -> Void>
source/NGio.hx:99: characters 30-45 : ... For optional function argument 'callback'
source/NGio.hx:85: characters 25-34 : () -> Void should be Null<io.newgrounds.LoginOutcome -> Void>
source/NGio.hx:85: characters 25-34 : ... For optional function argument 'callback'

**Reply by diskbeta** [reply]
this is what it showed after compiling

**Reply by gdandsm64fan123** [reply]
did u install the 4 gb of shit required and all flixel packages in the readme?

**Reply by diskbeta** [reply]
yes
